src/Init/Data/Iterators/Lemmas/Consumers/Collect.lean

```diff
@@ -49,24 +68,47 @@ theorem Iter.toList_toArray {α β} [Iterator α Id β] [Finite α Id] [Iterator
     it.toArray.toList = it.toList := by
   simp [toArray_eq_toArray_toIterM, toList_eq_toList_toIterM, ← IterM.toList_toArray]
 
+@[simp]
```

Comment:
Probably that doesn't even need to be `simp` I think? Should follow from the others?